add-chain/e2e_test.go

```diff
@@ -172,7 +172,6 @@ func checkConfigTOML(t *testing.T, testName, chainShortName string) {
 func cleanupTestFiles(t *testing.T, chainShortName string) {
 	paths := []string{
 		addressDir + chainShortName + ".json",
-		genesisSystemConfigDir + chainShortName + ".json",
```

Comment:
Should we remove the test artifacts written to `validationInputsDir` as well?

Reply:
Ideally yes, it's a bit difficult because the cleanup function doesn't have the chain id in scope at the moment. Will take another look to see if I can refactor it.